### PR TITLE
Add Db seed test

### PR DESCRIPTION
**Pull request by InsaneZein**
Seeded DB unit tests with data from quickstarts and helptopic files to write tests that make sure the db data matches what is in the files.

### CODE REVIEW

File: pkg/database/db_seed_testing.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package database
+
+// func VerifyNumberOfTags {
+// 	// initiate DB to read the real yml files into sqlite
+// 	// parse through the yml files and compare them to DB tables
+// 	//
+// }
```
Comment (Hyperkid123) [inline pkg/database/db_seed_testing.go:7]: No need for this empty file?

File: pkg/database/db_test.go [hunk 2]
```diff
@@ -75,9 +82,104 @@ func TestCreateQuickstartWithBundle(t *testing.T) {
 		DB.Find(&quickStarts)
 		DB.Model(&tag).Association("Quickstarts").Find(&quickStartsAssociations)
 		assert.Equal(t, dbTag.ID, tag.ID)
-		assert.Equal(t, 1, len(quickStarts))
+		assert.Equal(t, 6, len(quickStarts))
```
Comment (Hyperkid123) [inline pkg/database/db_test.go:85]: The number should be probably based on the number of quickstart yaml files. Content folks will be adding new files and we don't want them to touch test files.

File: pkg/database/db_test.go [hunk 3]
```diff
@@ -75,9 +82,104 @@ func TestCreateQuickstartWithBundle(t *testing.T) {
...
+	helpTopicsFiles, err := filepath.Glob(path + "/docs/help-topics/**/metadata.y*")
+	files := append(quickstartsFiles, helpTopicsFiles...)
+	t.Log(files)
+
+	t.Run("create DB seeding", func(t *testing.T) {
+		var quickStarts []models.Quickstart
+		DB.Find(&quickStarts)
+	})
+
+	// t.Run("tags match what is in DB", func(t *testing.T) {
```
Comment (Hyperkid123) [inline pkg/database/db_test.go:108]: unused code

**Comment by InsaneZein** [comment]
@Hyperkid123 I added the line that was missing, causing the failing test, and I've added the `make test` command to the docker file.

**Pull request merged**